telemetry/specs/INT.mdk

```diff
@@ -910,6 +913,190 @@ from (shim header length \* 4).
+techniques (e.g., ECMP, LAG) are used in the network. Note the Rep bits should
+be used judiciously (e.g., only for probe packets, not for every data packet).
+While we recommend that Rep bits be set only for probe packets, the INT
+architecture does not (and perhaps cannot) disallow use of the Rep bits for real
+data packets.
+    - 0: No replication requested.
+    - 1: Port-level (L2-level) replication requested. If the INT packet is
+forwarded through a logical port that is a port-channel (LAG), then replicate
+the packet on each physical port in the port-channel and send a single copy per
+physical port.
+    - 2: Next-hop-level (L3-level) replication requested. Forward the packet
+to each L3 ECMP next-hop valid for the destination address, with INT headers
+replicated in each forwarded copy.
+    - 3: Port-level and Next-hop-level replication requested.
+  - C (1b): Copy.
+    - If replication is requested for data packets, the INT Sink must be
+able to distinguish the original packet from replicas so that it can forward
+only original packets up the protocol stack, and drop all the replicas. The C
+bit must be set to 1 on each copy, whenever an INT hop replicates a packet.
+The original packet must have C bit set to 0.
+    - C bit must be set to 0 in the original packet by INT source
+  - E (1b): Max Hop Count exceeded.
+    - This flag must be set if a device cannot prepend its own metadata due to
+      the Remaining Hop Count reaching zero.
+    - E bit must be set to 0 by INT source
+  - M (1b): MTU exceeded
+    - This flag must be set if a device cannot add all of the requested metadata
+      because doing so will cause the packet length to exceed egress link MTU.
+      In this case, the device must not add any metadata to the packet, and set
+      the M bit in the INT header. Note that it is possible for egress MTU
+      limitation to prevent INT metadata insertion at multiple hops along a
+      path. The M bit simply serves as an indication that INT metadata was not
+      inserted at one or more hops and corrective action such as reconfiguring
+      MTU at some links may be needed, particularly when INT switches are not
+      participating in path MTU discovery. The M bit is not aimed at readily
+      identifying which switch(es) did not insert INT metadata due to egress MTU
+      limitation. In theory, if this does not occur at consecutive hops,
+      it may be possible for the monitoring system to derive which
+      switch(es) set the M bit based on knowledge of the network topology
+      and "Switch ID, Ingress port ID, Egress port ID" tuples in the INT
+      metadata stack.
+  - S (1b): Sink Node Specific Flag
+    - This flag is a INT Sink node specific flag to allow for the Sink nodes in the 
+      domain to process the metadata stack in a domain specific way.This flag must 
+      be set if a device, source node or transit node, wants to indicate to the sink
+      a domain specific handling of the metadata.
```

Comment:
Same comment as above. With "Domain Specific" expanding into "Reserved" (i.e. another 16 bits), the S bit is no longer needed in this first part of the header. There is plenty of room in the fields after the Domain Specific ID for these types of indications.

Reply:
Agree. Corrected